### PR TITLE
refactor: Disable Typescript suppressImplicitAnyIndexErrors option and correct errors

### PR DESCRIPTION
**Pull request by lukelooker**

### CODE REVIEW

File: packages/design-tokens/src/utils/paddingDefaultsHelper.ts [hunk 1]
```diff
@@ -66,12 +66,14 @@ export const paddingDefaultsHelper = (
     pt: py ? undefined : pt,
     px,
     py,
-  }
+  } as const
+
+  type responseKey = keyof typeof response
```
Comment (mdodgelooker) [inline packages/design-tokens/src/utils/paddingDefaultsHelper.ts:71]: Are we strict about capitalizing type alias?
```suggestion
  type ResponseKey = keyof typeof response
```

**Review by mdodgelooker — COMMENTED** [review]
Just a couple questions but overall looks good

File: packages/design-tokens/src/utils/paddingDefaultsHelper.ts (hunk 1 above)
Comment (lukelooker) [reply packages/design-tokens/src/utils/paddingDefaultsHelper.ts:71]: No but we probably should be :)

**Review by mdodgelooker — APPROVED** [review]
LGTM!

**Pull request merged**